ui/src/app/components/summary/summary.component.html

```diff
@@ -63,23 +63,23 @@
               error
             </mat-icon>
             <mat-icon
-              matTooltip="Warning"
+              matTooltip="Warning : Changes need to be made."
```

Comment:
Changes made because of differences in source and spanner capabilities.

Reply:
Updated